drafts/ecc-zf-split-plus-voting.rst

```diff
@@ -0,0 +1,103 @@
+* 2nd Halvening period - the 4-year period of time, roughly from October 2020 - October 2024, during which at most 5,250,000 ZEC will be minted
+* DF% - Dev Fund Percentage, the portion of newly minted ZEC in each block reserved for a dev fund
+
+Abstract
+========
+
+This proposal would allocate a 20% Dev Fund Percentage to be split evenly between the ECC and the Zcash Foundation during the 2nd Halvening period. This proposal aims to be simple to implement, without a single point of failure, and comes with mandates for transparency, accountability and a mandate to build a dev fund voting mechanism to be used during the 3rd Halvening period. This proposal is designed to strike a balance between ensuring that high quality development work can continue uninterrupted, with the need to further decentralize Zcash development funding as soon as possible.
+
+Motivation
+========
+
+Strengths of this proposal:
+
+1. Simple to implement. I would rather developers spend time scaling Zcash rather than implementing the dev funding mechanism.
+2. Developers will have several years to work on a great decentralized development funding voting mechanism.
+3. 20% is a large enough percentage that there should be enough dev funding for many different Zec price scenarios. To those who want to decrease this percentage: Overpaying for security is a gift to miners, to the detriment of every other stakeholder in the Zcash community. I will even make the strong statement that intentionally overpaying the miners for security is stealing from the other stakeholders.
+4. With two entities receiving funds there is no single point of failure.
+
+Requirements
+========
+
+1. DF% must be 20 percent, split evenly between company and foundation for years 5-8.
+2. Dev Fund must only to be spent on research, development and adoption of zcash
+3. A voting system for dev funding must be built and implemented after year 8.
+
+Specification
+========
+
+Voting System Requirements
+-----------------
+
+Here are the general properties of the mandated voting mechanism. I don’t want to specify the technical implementation details, since I believe this is a job suited for the engineers building this system.
+
+1. Voting should be private.
+2. Only Zec holders can vote.
+3. Voting should happen on-chain.
+4. In order to vote, you must lock your zec so it cannot be spent for a period of time. This is to force the voters to have ‘skin in the game’ and prevent someone nefarious from buying a lot of zec just before an election and then dumping it immediately after.
+5. Voters can choose how long to lock their zec, and their voting power is proportional to the time that the zec is locked. For example, someone who votes with 10 zec and locks it for 6 months would have the same voting power as someone who votes with 20 zec and locks it for 3 months. Of course there must be a maximum lock time, perhaps a year, to prevent anyone from getting ‘infinite’ voting power by locking their zec permanently.
```

Comment:
Locking funds into the *future* is problematic from a security perspective. If the owner's wallet is compromised, or a security-related movement of funds becomes desirable (e.g., consider the Sprout bug), then the owner would be unable to move their funds to protect them.

Additional considerations are discussed at https://forum.zcashcommunity.com/t/decentralized-participatory-voting-through-vesting/35088/3 .

So it may make sense to weaken these requirements leaving them up to future community discussion.